tests/bluetooth/bsim_bt/bsim_test_mesh/tests_scripts/heartbeat/sub_cb_api.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Copyright 2021 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Run unicast heartbeat subscription to a device
+RunTest mesh_heartbeat_sub_cb_api \
+	heartbeat_publish_unicast \
+	heartbeat_publish_unicast \
+	heartbeat_subscribe_unicast
+
+# Run all nodes heartbeat subscription to a device
+RunTest mesh_heartbeat_sub_cb_api_all \
```

Comment:
Please only put one RunTest in each test script, unless the test requires that these cases are executed in sequence.